src/test_typing_extensions.py

```diff
@@ -1211,7 +1211,7 @@ def test_illegal_parameters_do_not_raise_runtime_errors(self):
         # Type checkers should reject these types, but we do not
         # raise errors at runtime to maintain maximum flexibility
         Literal[int]
-        Literal[Literal[1, 2], Literal[4, 5]]
```

Comment:
The autofix is wrong but the comment above is also wrong; this is legal. See the example in the spec `Literal[Literal[Literal[1, 2, 3], "foo"], 5, None]`.

Reply:
The other ones in the list are illegal, just this one is allowed.

Reply:
oops I didn't look carefully enough

Reply:
fixed in https://github.com/python/typing_extensions/pull/628/commits/e4702d6b30e62ea9255c3946cf54d6bce14d2d88